setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    name="blackbox",
+    version="0.0.1",
+    description="Tool for automatic backups of databases",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    author="Leon Sandoy",
+    author_email="heavysaturn@gmail.com",
+    url="https://github.com/lemonsaurus/blackbox",
+    license="MIT",
+    packages=find_packages(exclude=["tests", "tests.*"]),
+    classifiers=[
+        "License :: OSI Approved :: MIT License",
+        "Development Status :: 3 - Alpha",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.9",
+        "Framework :: AsyncIO",
+        "Topic :: Database",
+        "Topic :: System :: Archiving :: Backup"
+    ],
+    install_requires=[
+        "pyyaml~=5.3.1",
+        "requests~=2.25.1",
+        "boto3~=1.16.51",
+        "loguru~=0.5.3",
+        "jinja2==2.11.2",
+    ],
```

Comment:
So, since we're duplicating our dependency list now, we should probably have some comments both here and in the Pipfile explaining that you need to add them both places.

Alternatively, can we pass the Pipfile into this file so we don't need this but of duplication? keep it DRY!